src/jit/morph.cpp

```diff
@@ -14057,6 +14057,12 @@ GenTree* Compiler::fgMorphModToSubMulDiv(GenTreeOp* tree)
         denominator = fgMakeMultiUse(&tree->gtOp2);
     }
 
+    if (lvaLocalVarRefCounted)
```

Comment:
I don't see why the difference is germane. Either way the refcounts amount to the same. I agree that we should not be cloning them in the case that the original numerator is not a leaf node, though.